src/configs/settings.ts

```diff
@@ -1,6 +1,25 @@
-/** This is the main configuration module */
+/** This is the main configuration module
```

Comment:
@cKellyDesign curious if you think this docstring has good guidelines for the settings module

Reply:
@moshthepitt I think these are great rules to instate, for me it's a bit of a paradigm shift in moving dynamic functionalities (functions and other imports) out of this file and into helpers in more relevant directories.

Should we also allow Typings to be imported? Should Typings ever be defined here?

Reply:
@cKellyDesign I dont know what to do about typings :( - maybe we should just leave them here until we are clearer on what to do about them?  Of course any type defs that naturally belong in other places should be defined in those places.

Reply:
@moshthepitt What if we added `types.ts` files adjacent to `helpers.ts` files?

Reply:
> @moshthepitt What if we added types.ts files adjacent to helpers.ts files?

I think we should do this @cKellyDesign - I created an issue for this https://github.com/onaio/reveal-frontend/issues/469